index.d.ts

```diff
@@ -48,6 +48,34 @@ declare namespace prettyBytes {
 		```
 		*/
 		readonly binary?: boolean;
+
+		/**
+		Number of significant digits to display when calculating bytes or bits.
+
+		@default 3
+
+		```
+		import prettyBytes = require('pretty-bytes');
+
+		prettyBytes(1019, {digits: 3});
+		//=> '1.02 B'
```

Comment:
The example should show with and without the option set so users can compare. Same with the other option.